src/views/IndividualTeamView.vue

```diff
@@ -53,9 +104,20 @@ export default {
       loaded: false,
       error: false,
       errorMessage: '',
+      constants: teamConstants,
```

Comment:
can this just be called teamConstants too

Reply:
Ah forgot to change that